src/inference/keyEndpoint.ts

```diff
@@ -107,16 +107,26 @@ export const key = (
   }
 
   // Check kid
-  let kid = serviceRequest.body?.["kid"];
+  let kidString = serviceRequest.query?.["kid"];
```

Comment:
let kidString = serviceRequest.body?.["kid"] || serviceRequest.query?.["kid"];

This will keep compatibility with existing server